### DISCUSSION TITLE
unreal-engine/damage-1/

### DISCUSSION THREAD

**Discussion by giscus** [JasperDeLaat94/jasperdelaat-com#2] [General]
# unreal-engine/damage-1/

Damage in Unreal Engine tutorial 1: An introduction to damage.

https://jasperdelaat.com/unreal-engine/damage-1/

**Comment by JTuthill01** [comment]
I know you posted this almost a year ago, but I just came across it and wanted to send you a HUGE thank you. I'm trying to figure out the damage system in UE. The fact you added C++, is just such a win-win for me. I'm using C++ and not having to figure out how to convert this into code is huge. I wish you would finish out this series.....
